helmfile.d/snippets/defaults.yaml

```diff
@@ -27,11 +20,11 @@ environments:
             applicationSet:
               replicas: 1
```

Comment:
since pdb minReplicas is set to one then here replicas should be set to 2. Otherwise PDB will block smooth migration between nodes, right?

Reply:
correct. Removed. Can be configured when we use sharding per cluster